markdown.go

```diff
@@ -1082,6 +1089,21 @@ func expandTabs(out *bytes.Buffer, line []byte, tabSize int) {
 	}
 }
 
+// Extract indent finds the appropriate indentation for the next block
+// where line can be inside of a list.
+func extractIndent(line []byte, tabSize int) []byte {
```

Comment:
If we are going add these refactors, then please do so in a separate PR, 'cause there are a gazillion places where similar code is used to find the indentation.